### ISSUE TITLE
Create an enumeration that echos the IPTC synthetic media codes, plus a property pointing to it

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#3392]
As a task within workarea #3391, we should have a schema.org native enumeration matching the codes standardized at IPTC for synthetic media. Also a property to use them on a MediaObject.

IPTC docs:
 * https://www.iptc.org/std/photometadata/documentation/userguide/
 * https://www.iptc.org/std/photometadata/documentation/userguide/#_about_the_iptc_photo_metadata_user_guide
 * https://iptc.org/news/iptc-releases-draft-of-digital-source-type-vocabulary-to-support-synthetic-media/

See https://cv.iptc.org/newscodes/digitalsourcetype/ for details. Here's a copy of their current wording format for one term, see link for full version:

```
Concept ID (QCode) = digsrctype:positiveFilm, ID (URI) = http://cv.iptc.org/newscodes/digitalsourcetype/positiveFilm
--
Type: cpnat:abstract | created: 2008-01-01T01:00:00+00:00 | modified: 2010-12-15T21:54:47+00:00 | retired:
Name(en-GB): Digitised from a positive on film
Definition(en-GB): The digital image was digitised from a positive on a transparency or any other transparent medium
Member of scheme: http://cv.iptc.org/newscodes/digitalsourcetype/
```

As always there is the question of adapting vs leaving unchanged when bringing in external mappings. Having tried some variations, I propose this:

 * DigitalCaptureDigitalSource
 * NegativeFilmDigitalSource
 * PositiveFilmDigitalSource
 * PrintDigitalSource
 * MinorHumanEditsDigitalSource
 * CompositeCaptureDigitalSource
 * AlgorithmicallyEnhancedDigitalSource
 * DataDrivenMediaDigitalSource
 * DigitalArtDigitalSource
 * VirtualRecordingDigitalSource
 * CompositeSyntheticDigitalSource
 * TrainedAlgorithmicMediaDigitalSource
 * CompositeWithTrainedAlgorithmicMediaDigitalSource
* AlgorithmicMediaDigitalSource

All of these within IPTCDigitalSource which is a subtype of (for now, we can add more and more structure) MediaEnumeration. I will also move the MediaReview enumeration beneath there.

To reference this list, for now we can simply use a digitalSource property. Other designs could reference these, or these plus others, in more ambitious ways. Note that in schema.org we never write enumeration types, enumeration members or anything else but properties using an initial lower case letter. This is why "print" becomes "PrintDigitalSource". The "DigitalSource" also stops us "using up" very general terms when we mean something quite specific. This is more of an issue for us at Schema.org since we do not use explicit sub-namespacing mechanisms, whereas at IPTC these terms are within an area already scoped to "digital source types".

**Comment by choochatjumpathongcj** [comment]
GDPRSPDRESG GDPRESGGROUP

**Comment by aisaac** [comment]
Hi @danbri , lurking around as we're thinking of re-using this vocabulary... Is the list of values actually finalized? We saw the list in the production namespace, but this ticket is still open.
If it's still up for discussion, the vocabulary is useful and makes sense, but the naming of instances is a bit confusing: "DigitalSource" hints that everything is a digital source. I reckon this issue is in the naming of the original IPTC vocabulary, but they didn't pass it to the naming of the values themselves.

**Comment by bquinn** [comment]
FYI we've just released an update to the IPTC Digital Source Type vocabulary: https://iptc.org/news/newscodes-2024-q3-release-including-media-topics-and-digital-source-type-updates/

**Comment by aisaac** [comment]
@bquinn Thank you this is extremely useful for us to know! 

Now this makes more acute the issues brought "mirroring/altering" the IPTC names in Schema.org: a couple of terms we wanted to use have now much more interesting names and semantics in the original IPTC namespace. So we're probably going to use them until Schema.org catches up. 

(this said even then it would be a great tribute to Schema.org's contribution to interoperability at large. Without it, we'd never have figured out that IPTC would cover this very specific need we have!)

**Comment by MatthiasWiesmann** [comment]
I created a pull request with the new values (I left the renames alone). 

https://github.com/schemaorg/schemaorg/pull/3691

You can see a test instance here: 

https://iptc-dot-schemadotorg1.ew.r.appspot.com

**Comment by aisaac** [comment]
Thanks! Is there a specific Schema.org rule that would prevent from removing and renaming the retired and updated elements at https://iptc.org/news/newscodes-2024-q3-release-including-media-topics-and-digital-source-type-updates/ ?

**Comment by MatthiasWiesmann** [comment]
We can't really delete things, but we could mark them as superseded.

**Comment by bquinn** [comment]
We would support marking the retired terms as superseded. FWIW we don't remove terms in IPTC vocabularies either - there's nothing to stop someone from using a retired term. Our retirement of a term is just a recommendation.

**Comment by MatthiasWiesmann** [comment]
Fair enough, would it be OK if a handle both cases in distinct pull requests?

**Comment by fthobe** [comment]
@MatthiasWiesmann do you have an example of superseeded terms, I could make a PR.

**Comment by joshdaugherty** [comment]
http://cv.iptc.org/newscodes/digitalsourcetype/digitalCreation is recommended as the replacement for  http://cv.iptc.org/newscodes/digitalsourcetype/digitalArt.

So then something like `DigitalCreationDigitalSource` would supersede `DigitalArtDigitalSource`.

---

http://cv.iptc.org/newscodes/digitalsourcetype/humanEdits is recommended as the replacement for http://cv.iptc.org/newscodes/digitalsourcetype/minorHumanEdits.

So then something like `HumanEditsDigitalSource` would supersede `MinorHumanEditsDigitalSource`.
